src/components/play-pause/play-pause.js

```diff
@@ -59,7 +59,7 @@ class PlayPause extends Component {
     const {eventManager, player} = this.props;
     // $FlowFixMe.
     const playerContainer: HTMLDivElement = document.getElementById(player.config.ui.targetId);
-    eventManager.listen(player, player.Event.Core.PLAY, () => {
+    eventManager.listen(player, player.Event.UI.USER_CLICKED_PLAY, () => {
```

Comment:
so if you do player.play() programatically it wont focus on the component ? seems correct, but please confirm with product

Reply:
yes and it's already confirmed with product

Reply:
ok